scripts/exportAssets.py

```diff
@@ -118,33 +162,40 @@ def main(argv):
     global OUTPUT_DIR
     global FILE_PATHS_AND_CONTENTS
     global TYPE
-    global ID
+    global TITLE
```

Comment:
All of these globals are not "pythonic". Prefer passing as params and returning modified variables rather than modifying globals.